docs/source/model_cards/BERT.md

```diff
@@ -1,4 +1,4 @@
-# ![bert](https://storage.googleapis.com/docs.mosaicml.com/images/models/bert.gif) BERT
+# 🦭 BERT
```

Comment:
I have no opinion as long as it works. There weren't any puppets, street signs, or sessame seeds.

Reply:
Thoughts about 🔁? The B in bert stands for bidirectional

Reply:
Ravi I'm so proud 🥹
they learn so quickly 🥲
(I don't have any opinion. will let everyone else decide)

Reply:
> bert is now a seal because slackmoji gifs don't work in the sidebar. sorry @bandish-shah

Hey no worries we tried.  Maybe Sesame Street just needs to get it's emoji act together.